lib/li_matrix.c

```diff
@@ -660,11 +660,20 @@ li_frustum_spherical_from_cartesian(LIFRUSTUM *res, const LIFRUSTUM *f)
 }
 
 
+void
+li_distortion_set(LIDISTORSION* dist, double pps0, double pps1, double c0, double c1, double c2)
```

Comment:
Back to this.

I tried the simpler approach first (`r2max=0`). pc_distortpolyradial5 now yields a result:

```
# select pc_distortpolyradial5('BOX4D(0 0 0 0, 10 10 0 0)'::LIBOX4D, ARRAY[2047.0,2048.0], ARRAY[0.00000000004106804505990795,-0.0000000000000000009287940817219032]);                                 pc_distortpolyradial5                                  
----------------------------------------------------------------------------------------
 BOX4D(-0.571197499432718 -0.571476540712363 0 0,9.43584817307658 9.43343990883249 0 0)
(1 row)
```

`xmin` and `ymin` are negative numbers. Is this expected?

And this is what I get when attempting to undistort the distorted box:

```
# select pc_undistortpolyradial5('BOX4D(-0.571197499432718 -0.571476540712363 0 0,9.43584817307658 9.43343990883249 0 0)', ARRAY[2047.0,2048.0], ARRAY[0.00000000004106804505990795,-0.0000000000000000009287940817219032]);
WARNING:  li_box_transform_undistorsion: current implementation is not conservative (bbox of 8points)...
                                pc_undistortpolyradial5                                 
----------------------------------------------------------------------------------------
 BOX4D(0.570794273644651 0.571073117940614 0 0,0.570794273644651 0.571073117940614 0 0)
(1 row)
```

Please tell me what you think. I'd like to have you check this and modify the functions if needed. Can you please look at it?